lib/bolt/core/src/tasks/api.rs

```diff
@@ -63,10 +60,7 @@ pub async fn get_cluster_server_ips(
 			project_ctx.origin_api(),
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 59]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Base path was updated in fern, should mirror change here
`{}/admin/clusters/server_ips`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/644?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Good call, didn't consider this